curious/utils.py

```diff
@@ -1,10 +1,13 @@
 import time
+import os
 
 # for development/debugging
 def report_time(f):
+  DEBUG = os.environ.get('DEBUG')
```

Comment:
if you're going to call it this, might as well have a more global setting for it, like in __init__.py